modules/mkr/hooks/useMkrDelegatedByUser.ts

```diff
@@ -34,9 +38,26 @@ export const useMkrDelegatedByUser = (
     };
   }
   const network = chainIdToNetworkName(chainId);
+
+  const fetchFromChain = (userAddress: string | undefined, voteDelegateAddress: string | undefined) => {
+    const contract = getEthersContracts<VoteDelegate>(
+      voteDelegateAddress as string,
+      abi,
+      chainId,
+      provider,
+      account,
+      true
+    );
+  
+    return contract.stake(userAddress as string);
+  };
+
```

Comment:
Won't this only return the MKR delegated directly by the EOA, and not the MKR delegated via the lockstake position? That's why we had to make the switch to the subgraph.